packages/lab/src/tokenized-input-next/TokenizedInputNext.tsx

```diff
@@ -0,0 +1,440 @@
+import {
+  Button,
+  ButtonProps,
+  makePrefixer,
+  NecessityType,
+  StatusAdornment,
+  useDensity,
+  useForkRef,
+  useId,
+  useIsomorphicLayoutEffect,
+} from "@salt-ds/core";
+import {
+  ChangeEventHandler,
+  FocusEvent,
+  FocusEventHandler,
+  ForwardedRef,
+  forwardRef,
+  HTMLAttributes,
+  KeyboardEvent,
+  KeyboardEventHandler,
+  ReactEventHandler,
+  Ref,
+  SyntheticEvent,
+  TextareaHTMLAttributes,
+  useCallback,
+  useRef,
+  useState,
+} from "react";
+import {
+  TokenizedInputNextHelpers,
+  TokenizedInputNextState,
+  useTokenizedInputNext,
+} from "./useTokenizedInputNext";
+import { clsx } from "clsx";
+import { InputPill } from "./internal/InputPill";
+import { CloseIcon, OverflowMenuIcon } from "@salt-ds/icons";
+import { useWidth } from "./internal/useWidth";
+import { useResizeObserver } from "./internal/useResizeObserver";
+import { calcFirstHiddenIndex } from "./internal/calcFirstHiddenIndex";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import tokenizedInputCss from "./TokenizedInputNext.css";
+
+type ChangeHandler<Item> = (selectedItems: Item[] | undefined) => void;
```

Comment:
All our handlers should have events being passed back, is there an issue stopping us here?